docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,789 @@
+ * Verify changing back the threshold values to original one and check alarm warning changing into False
+
+
+### 1.4 Check DPU Console
+
+#### Steps
+ * Use serial port utility to access console for given DPU.
+ * Get the mapping of serial port to DPU number from platform.json file. 
+ * Get the number of DPU modules from PMON APIs - get_num_modules(). Test is to check for console access for all DPUs.
+
+#### Verify in
+ * Switch
+   
+#### Sample Output
+```
+On Switch: (shows connection to dpu-4 console and offset is 4 for this case)
+
+root@sonic:/home/cisco# cat /proc/tty/driver/serial 
+serinfo:1.0 driver revision:
+0: uart:16550A port:000003F8 irq:16 tx:286846 rx:6096 oe:1 RTS|DTR|DSR|CD|RI
+1: uart:16550A port:00006000 irq:19 tx:0 rx:0 CTS|DSR|CD
+2: uart:16550A port:000003E8 irq:18 tx:0 rx:0 DSR|CD|RI
+3: uart:16550A port:00007000 irq:16 tx:0 rx:0 CTS|DSR|CD
+**4: uart:16550 mmio:0x94040040 irq:94 tx:0 rx:0
+5: uart:16550 mmio:0x94040060 irq:94 tx:20 rx:68
+6: uart:16550 mmio:0x94040080 irq:94 tx:0 rx:0
+7: uart:16550 mmio:0x940400A0 irq:94 tx:0 rx:0
+8: uart:16550 mmio:0x940400C0 irq:94 tx:0 rx:0
+9: uart:16550 mmio:0x940400E0 irq:94 tx:0 rx:0
+10: uart:16550 mmio:0x94040100 irq:94 tx:0 rx:0
+11: uart:16550 mmio:0x94040120 irq:94 tx:0 rx:0**
+12: uart:16550 mmio:0x94040140 irq:94 tx:0 rx:0 CTS|DSR
+13: uart:16550 mmio:0x94040160 irq:94 tx:0 rx:0 CTS|DSR
+14: uart:16550 mmio:0x94040180 irq:94 tx:0 rx:0 CTS|DSR
+15: uart:16550 mmio:0x940401A0 irq:94 tx:0 rx:0 CTS|DSR
+
+root@sonic:/home/cisco# /usr/bin/picocom -b 115200 /dev/ttyS8
+picocom v3.1
+
+port is        : /dev/ttyS8
+flowcontrol    : none
+baudrate is    : 115200
+parity is      : none
+databits are   : 8
+stopbits are   : 1
+escape is      : C-a
+local echo is  : no
+noinit is      : no
+noreset is     : no
+hangup is      : no
+nolock is      : no
+send_cmd is    : sz -vv
+receive_cmd is : rz -vv -E
+imap is        : 
+omap is        : 
+emap is        : crcrlf,delbs,
+logfile is     : none
+initstring     : none
+exit_after is  : not set
+exit is        : no
+
+Type [C-a] [C-h] to see available commands
+Terminal ready
+
+sonic login: admin
+Password: 
+Linux sonic 6.1.0-11-2-arm64 #1 SMP Debian 6.1.38-4 (2023-08-08) aarch64
+You are on
+  ____   ___  _   _ _  ____
+ / ___| / _ \| \ | (_)/ ___|
+ \___ \| | | |  \| | | |
+  ___) | |_| | |\  | | |___
+ |____/ \___/|_| \_|_|\____|
+
+-- Software for Open Networking in the Cloud --
+
+Unauthorized access and/or use are prohibited.
+All access and/or use are subject to monitoring.
+
+Help:    https://sonic-net.github.io/SONiC/
+
+Last login: Fri Jan 26 21:49:12 UTC 2024 from 169.254.143.2 on pts/1
+admin@sonic:~$ 
+admin@sonic:~$ 
+Terminating...
+Thanks for using picocom
+root@sonic:/home/cisco#
+
+```
+#### Pass/Fail Criteria
+ * Verify Login access is displayed.
+ * cntrl+a and then cntrl+x to come out of the DPU console.
+
+
+### 1.5 Check midplane ip address between NPU and DPU 
```

Comment:
This test case is not listed in the table "Objectives of Test Cases".

Reply:
Added it.